conf/docker-aio/1prep.sh

```diff
@@ -4,8 +4,9 @@
 # this was based off the phoenix deployment; and is likely uglier and bulkier than necessary in a perfect world
 
 mkdir -p testdata/doc/sphinx-guides/source/_static/util/
-cp ../solr/7.3.1/schema.xml testdata/
+cp ../solr/7.3.1/schema*.xml testdata/
 cp ../solr/7.3.1/solrconfig.xml testdata/
+cp ../solr/7.3.1/updateSchemaCMB.sh testdata/
```

Comment:
Throughout this pull request, would it make more sense to use "MDB" for "metadata block" rather that "CMB" for "custom metadata block"? I say this because these scripts and such are dealing generically with metadata blocks rather than only custom metadata blocks. Custom metadata blocks are an optional feature of Dataverse but there are 6 metadata blocks that are enabled out of the box and 1 (citation) that is required.

Reply:
Done in 555d1cea